Issue #2297 (#3823)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/hello-for-business/hello-identity-verification.md

| Windows Server 2016 AD FS with [KB4088889 update](https://support.microsoft.com/help/4088889) | Windows Server 2016 AD FS with [KB4088889 update](https://support.microsoft.com/help/4088889) |
| AD FS with Azure MFA Server, or</br>AD FS with 3rd Party MFA Adapter | AD FS with Azure MFA Server, or</br>AD FS with 3rd Party MFA Adapter |
| Azure Account, optional for Azure MFA billing | Azure Account, optional for Azure MFA billing |

>[!IMPORTANT]

> > On windows on-premises deployment, the Windows Server Domain Controller **must be** at least 2016 and once per domain.

Reviewer: ```suggestion
> Windows On-premises Deployments **requires** that the Windows Server Domain Controller is **at least** version 2016 and once per domain.
```
Sorry for not completing the edit, but I struggle too much to understand what "**once per domain**" refers to.
Does it mean that you need _at least_ 1 instance of the DC, or is it that you cannot have more than 1 per domain?

Author: Got it I've modified that.

> > On windows on-premises deployment, the Windows Server Domain Controller **must be** at least 2016 and once per domain.

Reviewer 2: ```suggestion
>For Windows on-premises deployment, the Windows Server Domain Controller **must be** version 2016 or later, and must be deployed once per domain.
```

> >For Windows on-premises deployment, if you have several domains, at least one Windows Server Domain Controller 2016 is required for each domain.

Reviewer: ```suggestion
>For Windows on-premises deployment, if you have several domains, at least one Windows Server 2016 Domain Controller is required for each domain.
```
My suggestion is based on the fact that "Windows Server 2016" is a product name in itself, and that it is not necessarily a Domain Controller in every scenario, but in this case, it needs to be one.

> >For Windows on-premises deployment, at least one Windows Server Domain Controller 2016 is required for each domain in case you have several domains.

Reviewer 2: ```suggestion
>For Windows on-premises deployment, if you have several domains, at least one Windows Server Domain Controller 2016 is required for each domain.
```

> >For Windows-hello deployment, at least one Windows Server Domain Controller 2016 is required for each domain in case you have several domains. For more information, please visit this [planning guide](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-adequate-domain-controllers).

Reviewer 2: ```suggestion
> For Windows-hello deployment, if you have several domains, at least one Windows Server Domain Controller 2016 is required for each domain. For more information, see the [planning guide](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-adequate-domain-controllers).
```
